dlp/snippets/redact/redact.go

```diff
@@ -38,7 +38,7 @@ func redactImage(w io.Writer, projectID string, infoTypeNames []string, bytesTyp
 
 	client, err := dlp.NewClient(ctx)
 	if err != nil {
-		return fmt.Errorf("dlp.NewClient: %w", err)
+		return fmt.Errorf("dlp.NewClient: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.